workitemtype.go

```diff
@@ -15,7 +15,7 @@ var wellKnown = map[string]*models.WorkItemType{
 		Name: "system.workitem",
 		Fields: map[string]models.FieldDefinition{
 			"system.owner": models.FieldDefinition{Type: models.SimpleType{Kind: models.KindUser}, Required: true},
-			"system.state": models.FieldDefinition{Type: models.SimpleType{Kind: models.KindString}, Required: true},
+			"system.url":   models.FieldDefinition{Type: models.SimpleType{Kind: models.KindURL}, Required: true},
```

Comment:
Don't add all types here. This is the "live workitemtype".. Add a test case to validate all the different types :)